test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,160 @@
+        m2_pubkey = m2.execute("wg pubkey < private").strip()
+        m2.execute("ip link add dev wg0 type wireguard")
+        m2.execute(f"ip addr add {m2_ip}/24 dev wg0")
+        m2.execute("wg set wg0 private-key ./private")
+        m2.execute(f"wg set wg0 listen-port {m2_port}")
+        m2.execute("ip link set wg0 up")
+
+        # Peer 1 (client)
+        m1_port = 51820
+        m1_ip = "10.0.0.1"
+        b.click("button:contains('Add VPN')")
+        b.wait_visible("#network-wireguard-settings-dialog")
+        b.wait_visible("button:contains('Save'):disabled")
+
+        # Validate each field, enter the right value, and then proceed to the next field
+        #
+        # check private-key
+        b.set_input_text("#network-wireguard-settings-private-key-input", "incorrect key")
+        b.set_input_text("#network-wireguard-settings-addresses-input", m1_ip)
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('key must be 32 bytes base64 encoded')")
+        b.click("button:contains('Generate')")
+
+        # check public-key
+        b.wait_not_val("#network-wireguard-settings-dialog .pf-v5-c-clipboard-copy input", "")
+        m1_pubkey = b.val("#network-wireguard-settings-dialog .pf-v5-c-clipboard-copy input")
+
+        # check listen-port
+        b.set_input_text("#network-wireguard-settings-listen-port-input", "66000")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('out of range')")
+        b.set_input_text("#network-wireguard-settings-listen-port-input", "sometext")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Listen port must be a number')")
+        b.set_input_text("#network-wireguard-settings-listen-port-input", str(m1_port))
+
+        # check ip addresses
+        b.set_input_text("#network-wireguard-settings-addresses-input", "10.0.0.1/24/56")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Addresses is not formatted correctly')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "10.0.0")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address 10.0.0')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "ten.one")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address ten.one')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "10 1")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid address 10')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "1.2.3.4/")
+        b.click("button:contains('Save')")
+        b.wait_visible(".pf-v5-c-alert:contains('Invalid prefix or netmask')")
+        b.set_input_text("#network-wireguard-settings-addresses-input", "1.2.3.4  ,  5.6.7.8  1.2.3.4.5")
```

Comment:
This tests multiple addresses with errors, but it's missing multiple *valid* addresses, and making sure that they are all actually applied to the interface.

Reply:
I was settings multiple valid addresses [here](https://github.com/cockpit-project/cockpit/pull/19024/files#diff-1e9b8cca06a692aac8083ddc21489f716c3ead2ba9cfd7b88de882faab5dbdd4R91) but didn't check whether it was applied. Fixed it [here](https://github.com/cockpit-project/cockpit/pull/19024/files#diff-1e9b8cca06a692aac8083ddc21489f716c3ead2ba9cfd7b88de882faab5dbdd4R109)